### ISSUE TITLE
Node specific stats

### ISSUE THREAD

**Issue by Crypnotic** [PufferPanel/PufferPanel#943]
Add an ability to see stats regarding different nodes registered on the panel.

I believe a resource graph that shows what all servers on the node are using would be nice in helping to visualize what nodes need more resources/offloading

**Comment by disconsented** [comment]
I'm going to argue why reinvent the wheel here with how excellent [Netdata](https://www.netdata.cloud/) is.

**Comment by Crypnotic** [comment]
Really trying to avoid installing more programs when deploying nodes, but I'll give it a shot.

**Comment by LordRalex** [comment]
We can probably provide a very "high" level the resource usage, but we won't be able to provide nearly as much info that dedicated tools can do. 

Leaving a package link so we can use it for this type of data
https://github.com/shirou/gopsutil

**Comment by Crypnotic** [comment]
I was mainly looking to see the cpu & memory usage that the MC servers are pulling on the node

**Comment by reven** [comment]
I agree this would be super useful for nodes with more than a few servers running. Just a simple CPU and memory (similar to server graphs) would help give an idea if the node is doing ok or if it can take another server.
